### PR TITLE
VACMS-000: Update code to match db state (these changes have appeared in every export for the past month).

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Trying to remove some developer friction - these changes have been appearing in every export for a while, and it's a pain to remove them.

## QA steps
Visual sanity check

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/core.entity_form_display.media.document.default.yml [hunk 1]
```diff
@@ -66,6 +66,5 @@ hidden:
   field_media_in_library: true
   moderation_state: true
   path: true
-  revision_log_message: true
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.media.document.default.yml:69]: Do we really want this going awol?

File: config/sync/core.entity_form_display.media.document.media_library.yml [hunk 2]
```diff
@@ -68,6 +68,5 @@ hidden:
   field_media_submission_guideline: true
   moderation_state: true
   path: true
-  revision_log_message: true
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.media.document.media_library.yml:71]: AWOL?

File: config/sync/core.entity_form_display.node.regional_health_care_service_des.inline_entity_form.yml [hunk 3]
```diff
@@ -59,7 +59,6 @@ hidden:
   promote: true
   revision_log_message: true
   status: true
-  revision_log: true
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.regional_health_care_service_des.inline_entity_form.yml:62]: Not sure on this one.

**Review by swirtSJW — COMMENTED** [review]
Overall looks good.   Just need to double check a couple of revision_log_message going AWOL.

File: config/sync/core.entity_form_display.media.document.default.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.media.document.default.yml:69]: Yep. Has no impact on the media doc form. Here it is after import of this config:
image

File: config/sync/core.entity_form_display.node.regional_health_care_service_des.inline_entity_form.yml (hunk 3 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.node.regional_health_care_service_des.inline_entity_form.yml:62]: We don't add the revision log to inline entity forms.
Safe to let this one go.

**Review by ndouglas — APPROVED** [review]
All LGTM except that one that's `revision_log` and not `revision_log_message`, IDK about that one, but ❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️

**Comment by ethanteague** [comment]
> All LGTM except that one that's `revision_log` and not `revision_log_message`, IDK about that one, but

I tested them all after config import, and they don't change the way we are displaying the rev log on forms, @ndouglas :+1:

**Comment by ndouglas** [comment]
Behat fail is just a blip but I'll rerun it just for the sake of completeness.

**Comment by ndouglas** [comment]
Still getting test failures with Behat even after `/va/web/build`.  Not sure what that's about 🤔  I feel like it's happened a couple times recently in different contexts.

**Comment by ndouglas** [comment]
Third time's the charm 😠

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**